linodecli/plugins/obj/__init__.py

```diff
@@ -106,16 +132,28 @@ def list_objects_or_buckets(
             prefix = ""
 
         data = []
+        objects = []
+        sub_directories = []
         try:
-            response = client.list_objects_v2(
-                Prefix=prefix, Bucket=bucket_name, Delimiter="/"
-            )
+            if parsed.all_rows:
+                responses = client.get_paginator("list_objects_v2").paginate(
+                    Prefix=prefix, Bucket=bucket_name, Delimiter="/"
+                )
+            else:
+                response = client.list_objects_v2(
+                    Prefix=prefix, Bucket=bucket_name, Delimiter="/"
+                )
+                if response.get("IsTruncated", False):
+                    warn(parsed, TRUNCATED_WARNING_MSG)
```

Comment:
I wonder if it'd be useful for users to be able to configure the page size for these requests. This might also come in handy for testing since we wouldn't need to create a bucket with 1000+ objects to test pagination.

Just a thought, feel free to ignore :slightly_smiling_face:

Reply:
Haha, that make sense. Thank you for the feedback! For now you can try to pass in a `PageSize` parameter for testing. I will add it to the PR later.